### ISSUE TITLE
osu!lazer for Android would be crash when typing

### ISSUE THREAD

**Issue by pan93412** [ppy/osu#6364]
**Describe the bug:** 
When I typing in osu!direct on osu!lazer for Android, it would be crash.

To reproduce this bug:
1. Open _osu!direct_
2. Typing in the search bar, and stop typing for 600 miliseconds. The result for the incomplete query has been shown up, and continuing typing.
3. It will be crashed after a while, about 200-300 miliseconds.

**Screenshots or videos showing encountered issue:** 

**osu!lazer version:** 
2019.930.0 [here](https://github.com/ppy/osu/releases/download/2019.930.0/sh.ppy.osulazer.apk)

**Logs:** 
```log
android.runtime.JavaProxyThrowable: System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Cannot access a disposed object.
Object name: 'MobileAuthenticatedStream'.) ---> System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'MobileAuthenticatedStream'.
  at Mono.Net.Security.MobileAuthenticatedStream.StartOperation (Mono.Net.Security.MobileAuthenticatedStream+OperationType type, Mono.Net.Security.AsyncProtocolRequest asyncRequest, System.Threading.CancellationToken cancellationToken) [0x0024b] in <741678b475264c7699dc4fbbae79283a>:0 
   --- End of inner exception stack trace ---
  at osu.Framework.Platform.GameHost+<>c__DisplayClass92_0.<handleException>b__0 () [0x00000] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at osu.Framework.Threading.ScheduledDelegate.RunTask () [0x00013] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at osu.Framework.Threading.Scheduler.Update () [0x001e5] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at osu.Framework.Threading.GameThread.ProcessFrame () [0x00050] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at osu.Framework.Threading.InputThread.RunUpdate () [0x00000] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at osu.Framework.Platform.GameHost.<Run>b__113_1 (System.Object <p0>, osuTK.FrameEventArgs <p1>) [0x00011] in <a529c42470544f41b73f01f3a9f77d6f>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[osuTK.FrameEventArgs].invoke_void_object_TEventArgs(object,osuTK.FrameEventArgs)
  at osuTK.GameViewBase.OnUpdateFrame (osuTK.FrameEventArgs e) [0x00008] in <7484b4753167456b82f11e78c0669d25>:0 
  at osuTK.Android.AndroidGameView.UpdateFrameInternal (osuTK.FrameEventArgs e) [0x00009] in <7484b4753167456b82f11e78c0669d25>:0 
  at osuTK.Android.AndroidGameView.RunIteration (System.Threading.CancellationToken token) [0x00051] in <7484b4753167456b82f11e78c0669d25>:0 
  at osuTK.Android.AndroidGameView+<>c__DisplayClass58_0.<StartThread>b__2 (System.Object _) [0x00000] in <7484b4753167456b82f11e78c0669d25>:0 
  at Android.App.SyncContext+<>c__DisplayClass3_0.<Send>b__0 () [0x00000] in <2e948e55030c4156962c061303f0e210>:0 
  at Java.Lang.Thread+RunnableImplementor.Run () [0x00008] in <2e948e55030c4156962c061303f0e210>:0 
  at Java.Lang.IRunnableInvoker.n_Run (System.IntPtr jnienv, System.IntPtr native__this) [0x00009] in <2e948e55030c4156962c061303f0e210>:0 
  at (wrapper dynamic-method) Android.Runtime.DynamicMethodNameCounter.20(intptr,intptr)
---> (Inner Exception #0) System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'MobileAuthenticatedStream'.
  at Mono.Net.Security.MobileAuthenticatedStream.StartOperation (Mono.Net.Security.MobileAuthenticatedStream+OperationType type, Mono.Net.Security.AsyncProtocolRequest asyncRequest, System.Threading.CancellationToken cancellationToken) [0x0024b] in <741678b475264c7699dc4fbbae79283a>:0 <---

 at mono.java.lang.RunnableImplementor.n_run(Native Method)
 at mono.java.lang.RunnableImplementor.run(Unknown Source:0)
 at android.os.Handler.handleCallback(Handler.java:794)
 at android.os.Handler.dispatchMessage(Handler.java:99)
 at android.os.Looper.loop(Looper.java:176)
 at android.app.ActivityThread.main(ActivityThread.java:6662)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
```

**Comment by mouzedrift** [comment]
duplicate of #6264

**Comment by pan93412** [comment]
thanks for notifying, i will close it.
